### PR TITLE
Docs 9972 password rotation

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

File: modules/ROOT/pages/managing-your-account.adoc [hunk 1]
```diff
@@ -180,12 +180,21 @@ You can modify your contact information, change your password, and see a list of
 
 == Reset Your Password
 
+When you reset your password, your new password must be between 8 and 64 characters in length with at least one uppercase letter, one lowercase letter, and one number. Your password can contain only the following:
```
Comment (dwang8) [inline modules/ROOT/pages/managing-your-account.adoc]: i'm not sure we need this
```
Your password can contain only the following: 
** Upper and lower case characters
** Numbers
** Non-alphanumeric symbols
```
was there a reason why you added that in?

Comment (dwang8) [reply modules/ROOT/pages/managing-your-account.adoc]: can we replace 
```
Your password can contain only the following: 
** Upper and lower case characters
** Numbers
** Non-alphanumeric symbols
```

with

```
Your password can contain ASCII-standard characters only, and accents and accented characters aren't supported.
```

File: modules/ROOT/pages/managing-your-account.adoc [hunk 2]
```diff
@@ -180,12 +180,21 @@ You can modify your contact information, change your password, and see a list of
 
 == Reset Your Password
 
+When you reset your password, your new password must be between 8 and 64 characters in length with at least one uppercase letter, one lowercase letter, and one number. Your password can contain only the following: 
+
+** Upper and lower case characters
+** Numbers
+** Non-alphanumeric symbols
+
```
Comment (sarathecoubian) [inline modules/ROOT/pages/managing-your-account.adoc]:
```suggestion
```

File: modules/ROOT/pages/managing-your-account.adoc (hunk 1 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/managing-your-account.adoc]:
```suggestion
When you reset your password, your new password must be between 8 and 64 characters in length with at least one uppercase letter, one lowercase letter, and one number. Your password must contain only ASCII-standard characters.

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**